docs/docs/flows.mdx

```diff
@@ -247,8 +246,97 @@ flows:
 ```
 
 ### Asking for confirmation
-- TODO: include flow but also slot configuration from domain
-- TODO: include branching on the result of the user selection
+
+excerpt from flows.yml
+```yaml
+flows:
+  transfer_money:
+    description: This flow demonstrates asking for confirmation from the user.
+    steps:
+      - id: "0"
+        intent: transfer_money
+        next: "1"
+      - id: "1"
+        question: money
+        next: "2"
+      - id: "2"
+        question: confirm_transfer
+        next:
+          - if: confirm_transfer == "yes"
+            then: "3"
+          - else: "4"
+      - id: "3"
+        action: utter_transfer_success
+      - id: "4"
+        action: utter_transfer_cancel
+```
+
+excerpt from domain.yml that shows the slot configuration.
+```yaml
```

Comment:
```suggestion
```yaml-rasa title="domain.yml"
```

Reply:
can you also add a few lines about what the slots configuration does? no need to go into too much details